### PR TITLE
[#5451] incorrect dataset type in some labels

### PR DESCRIPTION
**Pull request by smotornyuk**
Fix a few labels missed in the previous pr(#5189) and add records for towncrier

### CODE REVIEW

File: changes/5189.misc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Dataset type is used when building URLs(eg. `/showcase/edit/NAME`
+intead of `/dataset/edig/NAME` when `dataset['type']=='showcase'`)
```
Comment (wardi) [inline changes/5189.misc:2]:
```suggestion
intead of `/dataset/edit/dataset` when `dataset['type']=='showcase'`)
```

File: ckan/templates/package/base.html [hunk 2]
```diff
@@ -21,6 +21,6 @@
     <li{{ self.breadcrumb_content_selected() }}>{% link_for dataset|truncate(30), named_route=pkg.type ~ '.read', id=pkg.id if is_activity_archive else pkg.name %}</li>
   {% else %}
     <li>{% link_for _(dataset_type.title()), named_route=dataset_type ~ '.search' %}</li>
-    <li class="active"><a href="">{{ _('Create Dataset') }}</a></li>
+    <li class="active"><a href="">{{ _('Create ' ~ dataset_type.title()) }}</a></li>
```
Comment (wardi) [inline ckan/templates/package/base.html:24]: this kind of string can't be picked up by `extract_messages` but I guess we've done the same in the earlier PR.

File: ckan/templates/package/snippets/stages.html [hunk 3]
```diff
@@ -20,9 +20,9 @@
 <ol class="stages {{ class }}">
   <li class="first {{ s1 }}">
     {% if s1 != 'complete' %}
-      <span class="highlight">{{ _('Create dataset') }}</span>
+      <span class="highlight">{{ _('Create ' ~ dataset_type) }}</span>
```
Comment (wardi) [inline ckan/templates/package/snippets/stages.html:23]: You used `.title()` above, what's the difference here?

File: ckan/templates/package/base.html (hunk 2 above)
Comment (wardi) [reply ckan/templates/package/base.html:24]: See my suggestions in #5337 I think we should do the same here

File: ckan/templates/package/snippets/stages.html (hunk 3 above)
Comment (smotornyuk) [reply ckan/templates/package/snippets/stages.html:23]: I'm just trying to leave labels as they were before. In previous example **Dataset** was originally in title-case and here it was in lower-case before my change

**Comment by smotornyuk** [comment]
Hi, @wardi 

This PR is finished and can be reviewed. It does exactly the same things as #5337, but for datasets

**Comment by wardi** [comment]
@smotornyuk is there a front-end test that needs updating?

**Comment by smotornyuk** [comment]
Nope. CircleCI is failing because of some updates to the accessibility guide. Basically, it thinks that the contrast between text and background is not big enough:)
This test fails periodically on all the PRs and I think that @Zharktas has recently created fix for it

**Pull request merged**